assets/css/app.css

```diff
@@ -197,30 +221,18 @@ h2 {
   height: 19.99px;
 }
 
-.mt-0 {
-  margin-top: 0 !important;
-}
-
-.mb-0 {
-  margin-bottom: 0 !important;
-}
-
-.pb-0 {
-  padding-bottom: 0 !important;
-}
-
 .c-btn {
-  width: 175px;
+  width: auto;
   height: 48px;
   color: #fff;
   display: inline-block;
   text-decoration: none;
   font-style: normal;
   font-weight: 600;
-  font-size: 17px;
+  font-size: 15px;
   line-height: 24px;
   letter-spacing: 0.03em;
-  padding: 12px;
+  padding: 10px;
   transition: 0.2s all;
   background-color: #36b37f;
   border-radius: 4px;
```

Comment:
- [ ] One of the requirements of the project is to apply button interactions in hover and active states. the styling in the hover state is working but it styling in the active state is not working. kindly consider fixing it our because we need to stick to the Figma design as much as possible.

Reply:
![image](https://user-images.githubusercontent.com/3351180/214756424-18c96f38-3f6c-4a5c-a15d-98a4ab9c5eb7.png)

as you can see active state already there. please give me some clear explanation